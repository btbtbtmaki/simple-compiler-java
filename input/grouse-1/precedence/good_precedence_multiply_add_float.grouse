main {

  print 2.6 * 1.7 + 9.2 nl;
  print 2.9 * 4.8 + 9.5 nl;
  print 4.9 * 9.7 + 4.9 nl;
  print 6.3 * 4.6 + 6.3 nl;
  print 7.0 * 8.7 + 6.3 nl;
  print 4.7 * 7.9 + 6.0 nl;
  print 1.2 * 4.8 + 8.2 nl;
  print 6.1 * 8.9 + 2.9 nl;
  print 8.6 * 7.4 + 8.6 nl;
  print 1.4 * 4.8 + 8.0 nl;
  print 7.4 * 9.8 + 7.9 nl;
  print 2.8 * 8.0 + 1.4 nl;

  print 9.3 + 6.1 * 6.6 nl;
  print 1.0 + 8.4 * 9.0 nl;
  print 5.8 + 8.9 * 6.9 nl;
  print 8.2 + 5.5 * 6.7 nl;
  print 4.3 + 3.8 * 4.2 nl;
  print 7.2 + 4.3 * 4.8 nl;
  print 9.9 + 9.3 * 5.5 nl;
  print 5.9 + 6.1 * 2.4 nl;
  print 2.0 + 5.5 * 4.7 nl;
  print 2.2 + 2.7 * 2.2 nl;
  print 2.1 + 9.2 * 9.5 nl;
  print 3.9 + 9.2 * 6.3 nl;

  print 5.3 * 9.6 + 7.7 * 2.3 nl;
  print 8.9 * 7.8 + 3.5 * 9.5 nl;
  print 2.0 * 1.5 + 1.2 * 3.8 nl;
  print 7.7 * 9.9 + 9.8 * 7.8 nl;
  print 1.9 * 2.8 + 8.9 * 7.2 nl;
  print 9.9 * 7.5 + 2.5 * 6.3 nl;
  print 8.4 * 8.7 + 8.9 * 5.5 nl;
  print 9.7 * 2.7 + 7.2 * 8.9 nl;
  print 6.1 * 6.3 + 1.4 * 9.8 nl;
  print 4.3 * 2.4 + 3.6 * 3.4 nl;
  print 2.9 * 6.6 + 7.3 * 9.3 nl;
  print 1.9 * 3.6 + 8.4 * 3.4 nl;

  print 3.9 + 3.1 * 9.4 + 4.3 nl;
  print 7.2 + 8.1 * 8.3 + 8.0 nl;
  print 1.9 + 4.8 * 5.1 + 3.4 nl;
  print 9.1 + 6.6 * 8.3 + 5.9 nl;
  print 3.2 + 2.8 * 5.6 + 3.3 nl;
  print 1.8 + 6.6 * 8.3 + 4.0 nl;
  print 6.6 + 8.5 * 7.7 + 6.0 nl;
  print 9.3 + 5.6 * 9.2 + 9.8 nl;
  print 1.8 + 5.2 * 6.7 + 5.8 nl;
  print 8.5 + 6.0 * 6.9 + 5.3 nl;
  print 1.0 + 6.0 * 1.6 + 1.3 nl;
  print 6.5 + 6.1 * 8.0 + 9.8 nl;

}

