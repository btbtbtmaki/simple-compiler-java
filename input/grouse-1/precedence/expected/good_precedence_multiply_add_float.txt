13.62
23.42
52.43
35.28
67.2
43.13
13.96
57.19
72.24
14.72
80.42
23.8
49.56
76.6
67.21
45.05
20.26
27.84
61.05
20.54
27.85
8.14
89.5
61.86
68.59
102.67
7.56
152.67
69.4
90
122.03
90.27
52.15
22.56
87.03
35.4
37.34
82.43
29.78
69.78
22.18
60.58
78.05
70.62
42.44
55.2
11.9
65.1
